js/app/widgets/tableOfContents.js

```diff
@@ -415,7 +415,7 @@ const SectionButton = new Knowledge.Class({
         this.parent(params);
         let entry_class = Utils.get_element_style_class(TableOfContents, 'entry');
         this.get_style_context().add_class(entry_class);
-        section_title = section_title.toUpperCase();
+        // section_title = section_title.toUpperCase(); /* Don't want to delete yet, but this styling shouldn't be in the JS */
```

Comment:
I noticed you removed this line on a later commit, why not doing it here? Please try a to unify when possible.